broker/plugins/kubejobs/plugin.py

```diff
@@ -232,6 +251,17 @@ def errors(self):
         except redis.exceptions.ConnectionError:
             return ()
         return self.rds.lrange("job:errors", 0, -1)
+    
+    @set_timeout(1.0)
+    def persist_state(self):
+        atual_representation = self.__repr__()
+        if self.obj_representation != atual_representation:
+            self.persistence_obj.\
+                persist_state(self.app_id, atual_representation)
+            
+            self.obj_representation = atual_representation
+        self.persist_state()
```

Comment:
Will we have 1 thread for each job? If so, shouldn't the thread be killed as soon as the job is completed?
I think this line will throw a stack overflow error eventually, if the answer for the above question is yes, a `while <job is running condition>` with a `time.sleep(delay)` at the end should be fine.